packages/next-codemod/bin/upgrade.ts

```diff
@@ -23,7 +23,6 @@ const optionalNextjsPackages = [
   '@next/codemod',
   '@next/env',
   '@next/eslint-plugin-next',
-  '@next/font',
```

Comment:
You also need to ensure this is removed from user's `package.json`

Reply:
We've got codemod for it, should we validate within upgrade as well?
https://nextjs.org/docs/app/building-your-application/upgrading/codemods#132

Reply:
You could still have it in the dependency and you didn't apply the codemod before. Like the reported case is using `v14.0.3-canary.11`.

We could also add it into v15 codemod as well since the deprecation is made in v15, similar to the `experimental-edge`

Reply:
Hmm sounds like that should be apart of existing codemod process for `@next/font`.